### DISCUSSION TITLE
Thanks for this!

### DISCUSSION THREAD

**Discussion by saltmanz** [yegon666/CbzMage#4] [General]
Thank you! This is basically just what I needed. I've been using the download-the-AZW3-and-convert-with-Calibre method this entire time--which works fine--but your tool gives larger images with less compression, and it also manages to get around the two-page spread problems that plague other methods! (It's a shame Amazon generally cares so little about comic image quality, though: just as an example, I've got an old Uncanny X-Men comic whose cover image is a whopping 792px tall. I foresee a long drawn-out process of matching up the larger covers acquired through the old Comixology Backup extension with the higher-rez everything-else that this tool generates.)

**Comment by ToofDerling** [comment]
Thanks for the kind words, much appreciated. And yeah, getting as high quality backups as possible was very much on my mind.

**Comment by ToofDerling** [comment]
Would you mind telling me exactly which X-Men book you're talking about? I'll grab it and see if CbzMage is doing anything wrong.

**Reply by saltmanz** [reply]
It's Uncanny X-Men 269 (and most of the other issues around the same time), and it's actually 972px tall, I guess. Still super low-res considering the rest of the comic is 1920px tall. But I think it's just the way Amazon stores the covers; if I download the AZW3 file and open it in Calibre, the cover's the same size. Whereas the yanked-from-Comixology version was 1650px.
